gigasecond/test_gigasecond.py

```diff
@@ -0,0 +1,21 @@
+from gigasecond import gigasecond
+from datetime import datetime, timedelta
+
+seconds = timedelta(seconds=1000000000)
+
+
+def test_recently_apast_date_time():
+    assert gigasecond(datetime(2011, 4, 25)) == datetime(2011, 4, 25) + seconds
+
+
+def test_very_old_date_time():
+    assert gigasecond(datetime(1886, 2, 11)) == datetime(1886, 2, 11) + seconds
+
+
+# def test_todays_date():
```

Comment:
I wouldn't even test this...this is one of the hardest things to test.  The two `.now()` calls will be separated by a millisecond and will always fail. To get this test to work, you'd have to do:

````
current_time = datetime.now() # assures this current time is exactly the same
assert gigasecond(current_time == current_time + seconds
```